examples/emotiv_oscope_local.py

```diff
@@ -0,0 +1,64 @@
+"""
+Simple demonstration of streaming EEG data from an Emotiv device to a QOscilloscope
+viewer, for linux case.
+
+Both device and viewer nodes are created locally without a manager.
+"""
+
+from pyacq.devices.eeg_emotiv import Emotiv
+from pyacq.viewers import QOscilloscope
+import pyqtgraph as pg
+import os
+
+
```

Comment:
>     python3 emotiv_oscope_local.py '/dev/hidraw3'

Correct, but without quotes.

> on windows it's some ugly things like
> \?\hid#vid_1234&pid_ed02&mi_01#8&31721b0f&0&0000#{4d1e55b2-f16f-11cf-88cb-001111000030}

HM. I guess that is what it is, unless there's a way for us to simplify it for the user (maybe a way to automatically identify the emotiv?). Perhaps you can describe in the usage string how the user could find the device name in windows?